resolwe/storage/settings.py

```diff
@@ -2,13 +2,16 @@
 
 Used to provide s simple default configuration.
 """
+from pathlib import Path
+
 from django.conf import settings
 
+root = Path(getattr(settings, "PROJECT_ROOT", "/"))
```

Comment:
It would be better to rename the variable to `project_root`.

Reply:
OK.